parcels/particleset.py

```diff
@@ -156,7 +156,7 @@ def convert_to_array(var):
         for kwvar in kwargs:
             kwargs[kwvar] = convert_to_array(kwargs[kwvar])
             assert lon.size == kwargs[kwvar].size, (
-                '%s and positions (lon, lat, depth) don''t have the same lengths.' % kwargs[kwvar])
+                '%s and positions (lon, lat, depth) don''t have the same lengths.' % kwvar)
```

Comment:
Because `kwargs[kwvar]` is the vector of data, and `kwvar` is the name of the variable. The latter is more useful for debugging purposes.

Previously, the error would be something like`[0 0 0 0 ] and positions (lon, lat, depth) don't have the same lengths`. 
Now it is something like `p3 and positions (lon, lat, depth) don't have the same lengths.`